File: Vehicle-Entry-Exit-Backend-main/src/main/java/com/example/learn1/model/Vehicle.java
package com.example.learn1.model;

import lombok.Data;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

import java.util.List;

@Document(collection = "Vehicles")
@Data
public class Vehicle {
    @Id
    private String id;
    private String vehicleNumber;
    private String vehicleType;
    private String vehicleModelName;
    private List<String> vehicleImages; // Store URLs of images
    private String vehicleOwner;
    private String driverName;
    private String driverMobileNumber;
    private String empNumber;

}

File: VehicleEntryExit-Android-Fronted--main/app/src/main/java/com/example/vehicleentryexit/models/Vehicle.kt
package com.example.vehicleentryexit.models

data class Vehicle(
    val vehicleNumber: String,
    val vehicleType: String,
    val vehicleModelName: String,
    val vehicleImages: List<String>,
    val vehicleOwner: String,
    val driverName: String,
    val driverMobileNumber: String,
    val empNumber: String
)

